templates/service-template.yml

```diff
@@ -177,11 +177,6 @@ parameters:
   description: "YAML configuration for admin API endpoints authorization"
   value: "[{method: GET, roles: [kas-fleet-manager-admin-full, kas-fleet-manager-admin-read, kas-fleet-manager-admin-write]}, {method: PATCH, roles: [kas-fleet-manager-admin-full, kas-fleet-manager-admin-write]}, {method: DELETE, roles: [kas-fleet-manager-admin-full]}]"
 
-- name: ENTERPRISE_CLUSTER_REGISTRATION_ALLOWED_ORGANIZATIONS
```

Comment:
Once this is in, we'll need to update kas-installer.

/cc @MikeEdgar

Reply:
I've opened https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/273 to include this